blog/docs/articles/knative-backstage-plugins.md

```diff
@@ -0,0 +1,62 @@
+# Knative and Backstage
+
+**Author: Ali Ok, Principal Software Engineer @ Red Hat**
+
+## What's the vision?
+
+[Backstage](https://backstage.io/){:target="_blank"} is a platform for building developer portals. It provides a unified way to manage and visualize the different resources that developers use in their daily work. 
+
+However, Backstage is not your next generation Kubernetes dashboard. It is not designed to replace the Kubernetes dashboard or any other Kubernetes management tool.
```

Comment:
I think we should drop the use of However as it is there twice in the first paragraph here and may be try something like "While Backstage is not designed as a next generation Kubernetes Dashboard , it can visualize and partially manage Knative resources.  These resources would be read-only and focused on what's relevant to developers.